### DISCUSSION TITLE
Blank scan screen

### DISCUSSION THREAD

**Discussion by alexssdb** [capawesome-team/capacitor-mlkit#46] [Q&A] [answered]
Hello,

I have downloaded the project and I have tested it on my mobile phone and it works perfectly.

However, he implemented the functionality in a new app and everything works fine except the camera. When you perform the scan, the camera view does not appear in the modal.

I leave mobile screenshot.

Can anyone help me or any idea why?

Thank you!
![Screenshot_20230608_000037_io ionic starter](https://github.com/capawesome-team/capacitor-mlkit/assets/20858498/93b95410-eaff-42a6-9a87-aa060a754921)

**Comment by alexssdb** [comment, marked as answer]
Fixed.

I needed to add to the file
theme/variables.scss

Thanks!

  body.barcode-scanning-active {
    visibility: hidden;
    --background: transparent;
    --ion-background-color: transparent;
  }
  
  .barcode-scanning-modal {
    visibility: visible;
  }
  
  @media (prefers-color-scheme: dark) {
    .barcode-scanning-modal {
      --background: transparent;
      --ion-background-color: transparent;
    }
  }

**Reply by andrefnc4** [reply]
it happens the same to me, i have that code in the file 'theme/variables.scss' aswell and still doesn't work.
